### PR TITLE
Change content of the workshop for upcoming FOSSGIS

### PR DESCRIPTION
**Pull request by dmenneck**

### CODE REVIEW

**Comment by jansule** [comment]
In case this is ready for review, please remove the `WIP`

File: gitbook/geostyler/stepSix.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ werden. Das `defaultOlStyle` kann somit geparst werden.
 Der `GeoStyler` nimmt nun einen Stil an, jedoch verändert sich dieser beim Scrollen noch nicht. Diese 
 Verknüpfung mit den Daten erfolgt erst im kommenden Kapitel.
 
+***Aufgabe 1.***
+???
```
Comment (jansule) [inline gitbook/geostyler/stepSix.md]: Please remove, if there is no clear task

**Review by jansule — CHANGES_REQUESTED** [review]
Thanks for the work. Please note the change request.

**Review by jansule — APPROVED** [review]
Very nice. Thank you

File: gitbook/geostyler/stepSix.md (hunk 1 above)
Comment (dmenneck) [reply gitbook/geostyler/stepSix.md]: Task removed!

**Pull request merged**